README.md

```diff
@@ -10,6 +10,7 @@ to demonstrate how to use git and GitHub
 
 ## Spring 2022
 
+- [Adriana Beltran Andrade](https://github.com/abeltranandrade)
```

Comment:
Hey @abeltranandrade no shame in our game, but I think you accidentally listed yourself as part of Spring 2022, please move this to Fall 2022